scripts/benchmark/utils/chain_plumbing.py

```diff
@@ -81,19 +93,32 @@
 GenesisState = Iterable[Tuple[Address, Dict[str, Any]]]
 
 
-def get_chain(vm: Type[BaseVM], genesis_state: GenesisState) -> MiningChain:
+def get_chain(vm: Type[BaseVM], genesis_state: GenesisState) -> Iterable[MiningChain]:
 
-    chain = build(
+    # Chain uses MemoryDB
```

Comment:
Sorry, I probably wasn't clear with my previous comment. I meant we can drop usage of `MemoryDB` entirely and just go with `LevelDB` exclusively. The only reason `MemoryDB` was used at all is because:

a.) it was simple to start with
b.) it was thought that running benchmarks against `LevelDB` may be increase benchmark time too much

Turned out wrong, so it doesn't seem there's a good reason to keep `MemoryDB` at all here.